tests/lib/peers/PKI.test.ts

```diff
@@ -27,7 +27,7 @@ describe('PKI', () => {
       fs,
       keyManagerA
     )
-    while (!peerA.peerManager.serverStarted) {
+    while (!peerA.peerManager.peerServer.started) {
       await new Promise((resolve, reject) => {
```

Comment:
If you use this regularly, you should create a wrapper in your utils. So you get a `await utils.sleep()`.

Reply:
I am hoping to get rid of this when I do my xstate modelling, the node can't respond to network requests unless the server is started and the server cannot be started until the keynode is unlocked. All of these states and transitions will need to be properly modelled.

Reply:
I have added this to the utils for now, but hopefully we won't need this when we have a good state/transition model